src/third_party/catch/include/helpers.hpp

```diff
@@ -103,7 +103,10 @@
     auto collection_count_with_opts = libmongoc::collection_count_with_opts.create_instance();    \
     auto collection_create_index = libmongoc::collection_create_index.create_instance();          \
     auto collection_find = libmongoc::collection_find.create_instance();                          \
-    auto collection_aggregate = libmongoc::collection_aggregate.create_instance();
+    auto collection_aggregate = libmongoc::collection_aggregate.create_instance();                \
+    auto collection_get_name = libmongoc::collection_get_name.create_instance();                  \
+    collection_get_name->interpose([](mongoc_collection_t*){ return "dummy_collection"; });       \
+    auto collection_rename = libmongoc::collection_rename.create_instance();
```

Comment:
Right, that's what the integration test in `src/mongocxx/test/collection.cpp` would be for, running the operation against a db.  I think there's still validity to testing both the interface and the end-to-end behavior of the call.